### ISSUE TITLE
Fail on bad input

### ISSUE THREAD

**Issue by Licenser** [iocage/iocage#67]
Make sure to follow and check these boxes before submitting an issue! Thank you.

- [*] Supply `iocage --version`
`0.9.7`
- [*] Supply the commands used, along with any steps to recreate it.
```
[root@freebsd ~/iocage]# iocage create uuid=53c36348-0919-11e7-a71f-13e39e840134 tag=fifo-test -r 11.0-RELEASE
```
- [*] Provide the output from the command you issues.
```
***
WARNING: uuid is not a valid property!
***

e6fa13bd-a4ce-445a-b78a-995c0ab63eab (fifo-test) successfully created!
```
- [*] Supply what you expected the result or output to be
When supplying 'bad' arguments failing I would expect the command to fail instead of create a partially incomplete jail.
- [*] Checked that the problem has not already been fixed on `master` if using
 a stable release.
This is on master `fe30adef3c31decf62a25ff3c97eb03370fd18bb`

**Comment by skarekrow** [comment]
I'm not sure what you're asking here. I supply a warning when you give an invalid property but allow the jail creation to continue. It is a warning, not an error, so it isn't fatal. Would you prefer this to be fatal?

**Comment by Licenser** [comment]
It's of cause preference to a certain point so my thought is:

If I supply an invalid property / option I most of the time did something stupid (and I do lots of stupid things, don't tell anyone ;), so lets take the example above. I wanted to pass a option that doesn't exist, I (here comes the preference into play) don't want the jail to be created without it, I want the chance to fix it and then do it right. 

For me it gets worse when it comes to typos (and I do even more of those) so I tried to set `ipv4_address="..."` which was wrong and it told me and still created the jail. Now I got to delete that again, fix it, create it again.

It gets absolutely problematic when iocage is called programatically, a tool (*cough cough* not that I'd be doing that ;) calling iocage will get a 'succes' `0` return for a command was called with illegal parameters.

While **I** prefer failing early and hard - might have to do with my erlangyness - there is always the compromise to either make a `--strict` or `--yolo` flag to change the default behaviour.
